osu.Game/Online/API/Requests/Responses/BeatmapSetOnlineNominations.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Newtonsoft.Json;
+
+namespace osu.Game.Online.API.Requests.Responses
+{
+    public struct BeatmapSetOnlineNominations
```

Comment:
Any reason this is a `struct` rather than a `class` like all other API response models?

Reply:
I was just following genre and language (should be in the `osu.Game.Beatmaps` namespace then, my bad):
https://github.com/ppy/osu/blob/06e73594f80855ffa9cb08195bf65b84b8854c15/osu.Game/Beatmaps/BeatmapSetOnlineGenre.cs#L6-L12

Reply:
Sure I guess.